packages/main/src/ToggleSpinButton.ts

```diff
@@ -62,6 +62,13 @@ class ToggleSpinButton extends ToggleButton {
 	 * Override of the handler in order to prevent button toggle functionality
 	 */
 	_onclick() {}
+
+	/**
+	 * Override of the getter in order to set proper role to the button
+	 */
+	get buttonAccessibleRole() {
```

Comment:
We could just use an @override tag and not explain any further.
Additionally we should also remove the `role="spinbutton"` attribute from the `ToggleSplitButton.hbs` as we have an override now, which does the job successfully.

Reply:
Done